### DISCUSSION TITLE
How to use graphql in magento 2?

### DISCUSSION THREAD

**Discussion by devcrafta** [community/community#43663] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello Everyone.
How to use graphql in magento 2?

**Comment by doyen001** [comment, marked as answer]
Hi @cutesquirrel519 .
Please check this url.
[https://www.sjinnovation.com/how-to-use-graphql-in-magento-2](https://www.sjinnovation.com/how-to-use-graphql-in-magento-2)
